zvariant/src/value.rs

```diff
@@ -985,6 +985,7 @@ mod tests {
                 (@mmn nothing, @mmmn just nothing))"
         );
 
+        #[cfg(unix)]
```

Comment:
The commit log here "zv: fix build on windows" is also summarizing the benefit and not the change.

Reply:
done

Reply:
ok